test/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+include(FetchContent)
```

Comment:
This is how we are doing this reliably: https://github.com/vgvassilev/clad/blob/master/cmake/modules/CladGoogleTest.cmake